tests/testthat/test_part3_no4am.R

```diff
@@ -0,0 +1,36 @@
+library(GGIR)
+context("g.part3")
+test_that("Part 3 runs fine on a short file not containing a night", {
+  skip_on_cran()
+  #=======================
+  # Create a test file that doesn't contain a night.
+  # More specifically, it should not contan a 4am, since it's a magic time used in g.sib.det
+  create_test_acc_csv(Nmin=1000, starts_at_midnight = FALSE)
+  fn = "123A_testaccfile.csv"
+  dn = "output_test"
+  if (file.exists(dn)) unlink(dn, recursive = TRUE)
+  # g.part3 modules report errors and warnings only by printing them out to the console,
+  # so these are not true errors and warnings that would get caught by the testing module.
+  # Instead, grab the console output and parse out any errors we should know about.
+  # 
+  # Note: make sure to call GGIR() with do.parallel = FALSE,
+  # otherwise not all console output will be captured.
+  out = capture.output(GGIR(mode = c(1:3), datadir = fn, outputdir = getwd(), studyname = "test", f0 = 1, f1 = 1,
+                                    do.report = c(), visualreport = FALSE, viewingwindow = 1,
+                                    overwrite = TRUE, do.parallel = FALSE, minimumFileSizeMB = 0), 
+                       type = c("output"))
+  
+  # There shouldn't be any errors or warnings reported, so if there are any, throw an error.
+  err_idx = grepl("Errors and warnings for", out)
```

Comment:
Minor concern that I realized when testing this. I actually get a warning in part 3 when testing this which is not captured here:
```
Warning: time window shorter than 5 hours which makes it impossible to identify L5
```
The warning is not captured because it is printed to the console using `cat("Warning...")` instead of `warning()`. I already discussed this with Vincent and the proposal for fixing this would be changing the following lines in the function `g.sib.det`:

- line 230 should be removed (`cat("Warning...")`)
- line 231 should be kept as it is (`L5 = 0`)
- line 237 should state `L5 = 0` instead of `L5 = c()`.

Reply:
Thank you for catching this warning! I totally missed it because it got partially covered up by the `[ FAIL 0 | WARN 0 | SKIP 0 | PASS 1 ]` line!
I made the change you suggested.